### ISSUE TITLE
Reproducing results for qm7 dataset

### ISSUE THREAD

**Issue by arunppsg** [deepchem/deepchem#2776]
## ❓ Questions & Help

I was trying to reproduce the tests for qm7 dataset as in [test_qm7_multitask](https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/tests/test_molnet.py#L51), but I was not able to obtain R2-score > 0.75. The dataset is QM7 dataset, loaded with CoulombMatrix featurizer and stratified splitting. The task is a regression task.
 
I tried different models: MultitaskTransformRegressor, MultitaskRegressor, DTNN with varying training epochs and hyperparameters (like layer sizes) etc but the R2-score on train dataset wavered between 0.2 - 0.3 and also the R2-score on test set was 0.19-0.25 most of the times. I tried a Feed Forward neural network (implemented in pytorch and used dc.models.TorchModel API) but again got similar kind of results.

I am not sure why the model was not able to learn well. Any pointers on improving the benchmarking results will be helpful.
